ArgoTests/Models/URL.swift

```diff
@@ -0,0 +1,14 @@
+import Argo
+import Foundation
+
+extension URL: Decodable {
+  public typealias DecodedType = URL
```

Comment:
Is this actually needed? AFAIK we haven't tried removing it since `static` was added as a synonym for `final class`.